Eco-Map/eco-city-guide
# Feature / cleaning back end

## Task
> [!TIP]
> _Task description :_  
> This pull request clean the backend, relation for entities and start the first test 

> [!CAUTION]
> _Issue :_
> This PR don't close Issue 

## New dependencies ? ⚠️

> _Do we need to install a new depencency ? :_
- [x] yes &rarr;  `npm i interface-forge ` `npm i @faker-js/faker` `npm i ts-node` into back-end repertory

## How to Test 🧪
Launch the project with `make run` (maybe the dependencies need to be install manually)

Use command `make insert-mocks` for launch the mock entities and persist data in your db

After check your postgres Database for see the 20 places dynamical generated and 3 users
